### ISSUE TITLE
Issue with visibility of "About Us" page heading and button text.

### ISSUE THREAD

**Issue by arshiedu** [janavipandole/Hacktoberfest2025-Foodie#289]
Type of change: Bug Fix

The heading text is not visible in light mode and button text is also not visible in light as well as dark mode due to low colour contrast.

Fixes: (as shown in the image ready to raise PR)
I've fixed the bug can you please assign this to me.

I really want to contribute to this repo under Hacktoberfest2025.
May you please assign me this task.

<img width="1512" height="389" alt="Image" src="https://github.com/user-attachments/assets/e7a95da2-61f3-444c-8fb2-248de2ef2cc5" />

<img width="1512" height="357" alt="Image" src="https://github.com/user-attachments/assets/26351cda-c9b0-4e9c-8415-7f3abdd33870" />

<img width="1512" height="237" alt="Image" src="https://github.com/user-attachments/assets/0fe57703-59f2-4ece-822b-abfad12e88f5" />

<img width="1512" height="234" alt="Image" src="https://github.com/user-attachments/assets/ce289869-0fff-4e6d-bc13-5c01b27d314f" />

<img width="1512" height="561" alt="Image" src="https://github.com/user-attachments/assets/8690456d-e8f9-4abc-b537-1ca396c120d5" />

<img width="1512" height="543" alt="Image" src="https://github.com/user-attachments/assets/402f4885-e3a5-49e7-8800-b3a71d922371" />

**Comment by janavipandole** [comment]
no present
